app/controllers/sessions_controller.rb

```diff
@@ -10,14 +10,19 @@ def create
       render :new
     elsif user.authenticate(params[:password])
       session[:user_id] = user.id
-      flash[:success] = "Welcome #{user.user_detail.name}"
+      flash[:success] = "Logged in as #{user.user_detail.name}"
       dynamic_redirect
     else
       flash[:error] = "Invalid Credentials"
       render :new
     end
   end
 
+  def destroy
+    session[:user_id] = nil
```

Comment:
nice job proactively adding this, I will add the test for US 16 after the git fire is extinguished